README.md

```diff
@@ -113,8 +113,20 @@ your local instance. Here is one approach for running a local server:
 - Set up the web hook following the instructions in [Enabling a
   Repo](#enabling-a-repo), substituting your local Highfive IP address
   or domain name and port (if necessary).
-
-Here are couple things to watch out for:
+- Obtain an OAuth token. In the account you are creating the token in,
+  go to https://github.com/settings/tokens. Grant access to the repo
+  scope.
+- Put the authorization information obtained in the previous step into
+  a file named config in the top of the repository (i.e., the
+  directory containing this file). Here's a template of what it should
+  look like:
+   ```
+   [github]
+   user: OAUTH_TOKEN_USER
+   token: OAUTH_TOKEN
+   ```
+
```

Comment:
Looks good. Could you add a warning that users should not make the OAuth token public (including checking it in to a public repo, and further including in the Git history even if it is later removed).

Reply:
Absolutely.

Reply:
I added the following to below the config file example:
> _Do not check in this file or commit your OAuth token to a repository in any other way. It is a secret._